gcn/notices/ep/wxt/alert.schema.example.json

```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "https://gcn.nasa.gov/schema/main/gcn/notices/ep/wxt/alert.schema.json",
+  "mission": "EP",
+  "instrument": "WXT",
+  "trigger_time": "2024-03-01T21:46:05.13Z",
+  "ra": 120,
+  "dec": 40,
+  "ra_uncertainty": [0.02],
```

Comment:
@Vidushi-GitHub, could we please permit this to be a single number instead of a list?

Reply:
It's kept generic purposely, to accommodate elliptical, rectangle uncertainties. Ofcourse, we can keep it flexible (number/string) using Object type.

Reply:
Please open a PR to support a union of either a list or a single number.